### PR TITLE
feat(frontend): implement BTC wallet address component

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

The goal is to display BTC addresses in the way as it's done for other networks.

Desktop:
Screenshot 2024-09-25 at 11 21 03

Mobile:
Screenshot 2024-09-25 at 11 21 16

### CODE REVIEW

File: src/frontend/src/btc/components/core/BtcWalletAddress.svelte [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	const address = isNetworkIdBTCMainnet($networkId) ? btcAddressMainnet : btcAddressTestnet;
+</script>
+
+<div>
+	<label class="block text-sm font-bold" for="btc-wallet-address"
+		>{$i18n.wallet.text.wallet_address}:</label
+	>
+
+	<output id="btc-wallet-address" class="break-all"
+		>{shortenWithMiddleEllipsis({ text: $address ?? '' })}</output
```
Comment (peterpeterparker) [inline src/frontend/src/btc/components/core/BtcWalletAddress.svelte:19]: Using the default shortener with a default `splitLength` of 7 is also fine for bitcoin?

File: src/frontend/src/btc/components/core/BtcWalletAddress.svelte [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+	import { isNetworkIdBTCMainnet } from '$icp/utils/ic-send.utils.js';
+	import Copy from '$lib/components/ui/Copy.svelte';
+	import { btcAddressMainnet, btcAddressTestnet } from '$lib/derived/address.derived';
+	import { networkId } from '$lib/derived/network.derived';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { shortenWithMiddleEllipsis } from '$lib/utils/format.utils';
+
+	// Regtest and Testnet BTC wallets have the same address
+	const address = isNetworkIdBTCMainnet($networkId) ? btcAddressMainnet : btcAddressTestnet;
```
Comment (peterpeterparker) [inline src/frontend/src/btc/components/core/BtcWalletAddress.svelte:10]: Same comment as in previous PR, should we default to mainnet?

Comment (DenysKarmazynDFINITY) [reply src/frontend/src/btc/components/core/BtcWalletAddress.svelte]: Done.

File: src/frontend/src/btc/components/core/BtcWalletAddress.svelte (hunk 1 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/btc/components/core/BtcWalletAddress.svelte]: Yes, as we can see in the receive modal where it's also implemented with the default `splitLength`, it works fine for BTC addresses:
Screenshot 2024-09-25 at 13 10 23

**Pull request merged**